src/index.html

```diff
@@ -3,11 +3,357 @@
+          </article>
+
+          <article class="explore-block">
+            <img
+              src="./images/photos/explr-3.png"
+              alt="pic"
+              class="explore-image"
+            >
+
+            <h4 class="h4">Virtual group tours</h4>
+            <p class="p-roboto">Request a guided tour, reserve admission and lead your own small group, or request a live virtual tour led by a Museum guide.</p>
+          </article>
+
+        </div>
+
+      </section>
+
+      <section class="contact" id="contact">
+        <h2 class="h2">Contact us</h2>
+
+        <div class="info-wrapper">
+
+          <div class="adress-block block-1">
+            <strong class="strong">
+              The Met Fifth Avenue
+            </strong>
+
+            <p class="p5">Phone</p>
+
+            <a class="inter-color" href="tel:12125357710">
+              <strong class="strong">
+                +1 212-535-7710
+              </strong>
+            </a>
+
+            <p class="p5">Address</p>
+
+            <a
+              class="inter-color"
+              href="https://goo.gl/maps/Py6xeHdSGS3m4q1b7" target="_blank"
+            >
+              <strong class="strong">
+                1000 Fifth Avenue New York, NY 10028
+              </strong>
+            </a>
+          </div>
+
+          <div class="adress-block block-2">
+            <strong class="strong">
+              The Met Cloisters
+            </strong>
+
+            <p class="p5">Phone</p>
+
+            <a class="inter-color" href="tel:12125357710">
+              <strong class="strong">
+                +1 212-923-3700
+              </strong>
+            </a>
+
+            <p class="p5">Address</p>
+
+            <a
+              class="inter-color"
+              href="https://g.page/fort-tryon-park?share" target="_blank"
+            >
+              <strong class="strong">
+                99 Margaret Corbin Drive Fort Tryon Park New York, NY 10040
+              </strong>
+            </a>
+          </div>
+
+          <div class="form-wrapper adress-block block-3">
+            <form
+              action="#"
+              method="get"
+              class="form"
+              id="form"
+            >
+              <input
+                type="text"
+                name="name"
+                id="name"
+                placeholder="name"
+                class="form-field"
+                required
+              >
+
+              <input
+                type="email"
+                name="email"
+                placeholder="email"
+                id="email"
+                class="form-field"
+                required
+              >
+
+              <textarea
+                name="message"
+                id="message"
+                placeholder="message"
+                class="form-field"
+                required
+              ></textarea>
+
+              <button
+                type="reset"
```

Comment:
use [reset method](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset)

```suggestion
                type="submit"
```